### DISCUSSION TITLE
API Examples?

### DISCUSSION THREAD

**Discussion by Fxdli** [Itz-Hyperz/firewallgg#25] [Q&A] [answered]
How do I call the API?

**Comment by Itz-Hyperz** [comment, marked as answer]
We have our API examples listed here:
https://github.com/Itz-Hyperz/firewallgg/tree/main/examples

**Reply by Itz-Hyperz** [reply]
You can always feel free to add more as-well!

**Reply by Fxdli** [reply]
thank youuuuu
